runtime/str.go

```diff
@@ -699,6 +741,26 @@ func strRIndex(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	})
 }
 
+func strRJust(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	expectedTypes := []*Type{StrType, IntType, StrType}
+	if raised := checkMethodArgs(f, "rjust", args, expectedTypes...); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	width := toIntUnsafe(args[1]).Value()
+	fill := toStrUnsafe(args[2]).Value()
+
+	if numChars := len(fill); numChars != 1 {
+		return nil, f.RaiseType(TypeErrorType, fmt.Sprintf("rjust() argument 2 must be char, not str"))
+	}
+
+	if len(s) >= width {
+		return NewStr(s).ToObject(), nil
+	}
+
+	return NewStr(pad(s, width-len(s), 0, fill)).ToObject(), nil
```

Comment:
Seems like everything above this line is common to all the methods except for the name. Maybe factor it out into strJustDecodeArgs or something